packages/plugins/users-permissions/admin/src/pages/Roles/ListPage/index.js

```diff
@@ -130,7 +131,9 @@ const RoleListPage = () => {
         />
         <CustomContentLayout canRead={canRead} isLoading={isLoading || isLoadingForPermissions}>
           <Box paddingBottom={4}>
-            <Search />
+            <Row style={{ flexWrap: 'wrap' }}>
+              <Search />
+            </Row>
```

Comment:
@soupette got this from the admin Users/LIstPage so that at least there is some consistency 🤷‍♂️